packages/react-components/react-dialog/src/components/DialogSurface/__snapshots__/DialogSurface.test.tsx.snap

```diff
@@ -4,7 +4,7 @@ exports[`DialogSurface renders a default state 1`] = `
 <div>
   <div
     class="fui-DialogSurface"
-    data-tabster="{\\"deloser\\":{},\\"modalizer\\":{\\"id\\":\\"modal-9\\",\\"isOthersAccessible\\":false}}"
+    data-tabster="{\\"deloser\\":{},\\"modalizer\\":{\\"id\\":\\"modal-8\\",\\"isOthersAccessible\\":false}}"
```

Comment:
Why did this change? (It doesn't appear to be anything to worry about... but I wouldn't expect any snapshots to change from this PR)

Reply:
Answering my own question: It appears that it happened because DialogSurface.test.tsx doesn't reset test IDs after each test. Even though DialogSurface had disabled one of the conformance tests you're removing, it didn't disable the other, so there's still net one fewer test being run, which was presumably using a generated ID somewhere.

Reply:
Oops, good catch 👍 FYI @bsunderhus

Reply:
Thx for the heads up. I'll address that!